### ISSUE TITLE
Visual Keyboard hint sows up on mobile ios platform

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#508]
I just noticed I get the keyboard hint for the grab button when using my iphone.
The spoken content and hints delivered through VoiceOver are correct.
- Yellow Balloon, moveable
- PDOM Help text: Double tap and hold to grab Yellow Balloon. Lift to release.
- Upon first grab correct platform-specific help text: Yellow Balloon, Grabbed. At center of Play Area. Has no more negative charges than positive charges. Drag finger to move balloon. Lift to release.
- Upon subsequent grabs no help text: Yellow Balloon, Grabbed. {{position}}. {{comparative-charges}}. 

In addition to platform-specific help text, please make the visual keyboard platform specific as well. The visual keyboard hint should not appear on iOS (mobile platform).

**Comment by terracoda** [comment]
This should be part of the common-code mobile grab button.

**Comment by jessegreenberg** [comment]
This is related to the mobile grab-and-drag work, but we decided that this publication of BASE will not support this so I don't think this is blocking publication anymore and this is being deferred.

**Comment by terracoda** [comment]
Correct, we are deferring the publication of BASE with VoiceOver accessibility on iOS devices (i.e., mobile accessibility). In the interim, BASE is listed with mobile accessibility in the prototype section on the Accessible Simulations page.

**Comment by zepumph** [comment]
One thing to do here is to see if GFL on master is broken like this (since it has the grab drag interaction and has been published with mobile a11y). That would make it known if we need to adapt each new sim, or if something in common code is broken.

**Comment by zepumph** [comment]
We may want to move this to common code. Marking undeferred so that I can work on this to fix for friction.
